src/main/java/io/pinecone/clients/Pinecone.java

```diff
@@ -98,79 +149,242 @@ public IndexModel createServerlessIndex(String indexName, String metric, int dim
         return indexModel;
     }
 
-    // Minimal
+    /**
+     * Overload for creating a new pod index with environment and podType.
+     *
+     * <pre>
+     *     import io.pinecone.clients.Pinecone;
+     *
+     *     Pinecone client = new Pinecone.Builder("your-api-key").build();
+     *
+     *     client.createPodsIndex("my-index", 1536, "us-east4-gcp", "p1.x2");
```

Comment:
You might also want to show that you can use `sys.getenv` or whatever the call is for the environment etc

Reply:
Good call, I should do that for all of these tbh.